source/style-guide/information-types/prototypes/reference/server-params-style.txt

```diff
@@ -0,0 +1,129 @@
+.. _meta-reference-server-params:
+
+=======================
+Server Parameters Style
+=======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you document a new server parameter, include the following details 
+in addition to specific information about the new parameter. Document 
+these details in the order they appear in the following table:
+
+.. list-table::
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Documentation Detail
+     - Necessity
+     - Description
+
+   * - ``mongod`` / ``mongos`` Support
+     - Required
+     - Whether the parameter is supported on ``mongod``, ``mongos``, or both.
+       Use the ``|mongod-only|``, ``|mongos-only|``, or ``|both|`` substitutions
+       as appropriate.
+
+   * - Version the Parameter was Added or Changed In
+     - Conditional
+     - 
+       - For recent versions, include the version the parameter was added 
+         or changed in.
+       - For archived versions, delete the version added tag.
```

Comment:
[n] Not sure if list items are needed here, to me in a table it's a little jarring as the style is different from the other rows. I think each can just be a sentence on a new line within the cell.

Reply:
Good call. Updated!